src/index.html

```diff
@@ -6,20 +6,29 @@
   <body>
     <section>
       <h1>Howdy Partner</h1>
-      <section class="login-info">
+      <form class="login-info">
         <label class="user-inputs">Username:</label>
         <input class="user-inputs username" type="text" placeholder="please enter username">
         <label class="user-inputs">Password:</label>
         <input class="user-inputs password" type="text" placeholder="please enter password">
         <button class="user-inputs login-button">Log In</button>
-      </section>
+      </form>
       <img>
     </section>
     <section class="manager-display hidden">
-      Manager Display
+      <section class="manager-dashboard">
```

Comment:
HTML looks good. For better semantic HTML (and better accessibility scores), you might want to try to use articles and sections, rather than having sections inside of sections. But if you run lighthouse and get a good score, please ignore me! ;)